bcda/models/models_test.go

```diff
@@ -690,57 +451,6 @@ func (s *ModelsTestSuite) TestJobStatusMessage() {
 	assert.Equal(s.T(), "Completed", j.StatusMessage())
 }
 
-func (s *ModelsTestSuite) TestGetMaxBeneCount() {
```

Comment:
Test moved to [TestGetMaxBeneCount](https://github.com/CMSgov/bcda-app/pull/568/files#diff-f721bcd60ab3dc5c931f548ae66056f6R55)